code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm

```diff
@@ -119,7 +119,7 @@
 	gain_text = "Cursed land, cursed man, cursed mind."
 	desc = "Curse someone for 2 minutes of vomiting and major organ damage. Using a wirecutter, a spill of blood, a heart, left arm and a right arm, and an item that the victim touched  with their bare hands."
```

Comment:
Curse someone for 2 minutes of vomiting and major organ damage. Using wirecutters, a heart and an item that the victim touched with their bare hands.